index.html

```diff
@@ -115,6 +115,74 @@ <h4 class="heading heading_level-4">Подушка</h4>
             </div>
         </section>
 
+		<section class="services">
+			<div class="column-wrapper">
+				<div class="column-row">
+					<div class="column column_s-2 column_m-2 column_l-4">
+						<div class="services__item services__item_paddingRight">
```

Comment:
Названия классов должны быть в нижнем регистре. Для разделения слов используется `-`
```suggestion
						<div class="services__item services__item_padding-right">
```
А вообще здесь ты попытался подогнать конкретный текст, как на макете. Всегда надо допускать, что текст может быть другой и соответственно будет переносится по-другому. Так что модификатор здесь лишний. А чтобы текст правильно переносился, можно расставить неразрывные пробелы https://www.artlebedev.ru/typograf/